frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -134,6 +135,43 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                         checked={appearance?.shuffleQuestions}
                     />
                 </div>
+                <div className="mt-1">
+                    <LemonField name="survey_popup_delay" className="font-medium">
```

Comment:
nit: Don't use a lemonField here since the `name` in `LemonField` refers to an actual parameter on the survey which in this case doesn't exist. If you want a `LemonField` like handling, use `LemonField.Pure` instead